src/java/com/unifina/service/NotifyStorageNodeTask.java

```diff
@@ -0,0 +1,54 @@
+package com.unifina.service;
+
+import com.streamr.client.StreamrClient;
+import com.unifina.domain.Stream;
+import com.unifina.domain.StreamStorageNode;
+import com.unifina.domain.EthereumAddress;
+import org.apache.log4j.Logger;
+import java.util.Map;
+import java.util.HashMap;
+
+public class NotifyStorageNodeTask extends Thread {
```

Comment:
This class is not intended for subclassing. Lets make it final.
And its not intended for usage outside this package so lets make it package local.